koku/reporting/models.py

```diff
@@ -176,3 +177,31 @@
     "gcp_openshift_daily_resource_matched_temp": "ocp_source",
     "gcp_openshift_daily_tag_matched_temp": "ocp_source",
 }
+
+# These are cleaned during expired_data flow
+EXPIRE_MANAGED_TABLES = {
```

Comment:
These could be [MappingProxyType](https://docs.python.org/3/library/types.html#types.MappingProxyType) in order to make them immutable.